lua/toggleterm.lua

```diff
@@ -184,6 +184,80 @@ function M.exec(cmd, num, size, dir, direction, go_back, open)
   term:send(cmd, go_back)
 end
 
+--- @param selection_type string
+--- @param trim_spaces boolean
+--- @param terminal_id number
+function M.send_lines_to_terminal(selection_type, trim_spaces, terminal_id)
+  -- trim_spaces defines if we should trim the spaces from lines which are sent to the terminal
+  if trim_spaces == nil then trim_spaces = true end
+
+  -- If no terminal id provided fall back to the default
+  if terminal_id == nil then terminal_id = 1 end
+  terminal_id = tonumber(terminal_id)
+
+  vim.validate({
+    selection_type = { selection_type, "string", true },
+    trim_spaces = { trim_spaces, "boolean", true },
+    terminal_id = { terminal_id, "number", true }
+  })
+
+  -- Window number from where we are calling the function (needed so we can get back to it automatically)
+  local current_window = vim.api.nvim_get_current_win()
```

Comment:
This file already contains an alias for brevity where all api methods use the `api` alias created at the top of the file so all usages of `vim.api` could just be `api`. The same goes for `vim.fn` to `fn`

Reply:
done